lib/log.dart

```diff
@@ -30,3 +33,45 @@ bool debugLog(String message) {
   }());
   return true;
 }
+
+/// Display an error message.
+///
+/// This shows a [SnackBar] containing the message after app startup,
+/// otherwise logs it to the console.
+///
+/// See also: [ZulipApp._reportErrorToUserBriefly]
```

Comment:
This is a private method, and also kind of feels like an implementation detail. Is this the kind of information that callers need, to understand if `reportErrorToUserBriefly` is right for them, and what its expectations/guarantees are like? (Similarly with the dartdoc on `reportErrorToUserInDialog`.)

Reply:
The primary goal of including this line is to have a back link to where the setters get used, since it is not obvious. I think it would be more appropriate to exclude that bit from the dart doct, though, considering the point that it's some implementation detail.